### PR TITLE
Feature/regarch 64 related buildings

### PR DESCRIPTION
**Pull request by igor-kamil**
https://jira.sng.sk/browse/REGARCH-64

screenshot-www register-architektury local-2020-06-16-22-20-37-352

### CODE REVIEW

File: app/Models/Building.php [hunk 1]
```diff
@@ -182,7 +187,9 @@ public function getLngLatAttribute()
 
     public function toSearchableArray()
     {
-        return $this->toSearchableArrayWithTranslations();
+        $array = $this->toSearchableArrayWithTranslations();
+        unset($array['processed_images']);
+        return $array;
```
Comment (eronisko) [inline app/Models/Building.php:192]:
```suggestion
        return Arr::except($this->toSearchableArrayWithTranslations(), ['processed_images']);
```

Comment (eronisko) [reply app/Models/Building.php:192]: Interesting, these `getXXXAttribute`s get automatically included in `toArray()`?

File: app/Http/Controllers/BuildingController.php [hunk 2]
```diff
@@ -19,8 +19,26 @@ public function show($id, $slug, Request $request)
...
+                            'min_doc_freq' => 1,
+                        ]
+                    ]
+                ];
+            })
+            ->paginate(8);
+
     	return view('building.show', [
-            'building' => $building
+            'building' => $building,
```
Comment (eronisko) [inline app/Http/Controllers/BuildingController.php:40]: I'd consider using `compact` here

File: app/Models/Building.php (hunk 1 above)
Comment (igor-kamil) [reply app/Models/Building.php]: no. not automatically. if you want to, you have to add it into `$append`
https://github.com/SlovakNationalGallery/register-architektury/pull/38/files#diff-ea3ef69cca755b61049f093b9313922aR24

File: app/Http/Controllers/BuildingController.php (hunk 2 above)
Comment (igor-kamil) [reply app/Http/Controllers/BuildingController.php:40]: I guess more attributes are coming soon...

**Pull request merged**